### DISCUSSION TITLE
How to get set up coc.nvim language server for projects that use multiple repositories

### DISCUSSION THREAD

**Discussion by amandeepgautam** [neoclide/coc.nvim#5148] [General]
We have projects that have multiple repos. We use artifacts generated by other repo in the main project. I want to setup coc.nvm so that this all acts like one single project i.e. I can move around the code without knowing if it was from a different repo. Currently, the way it is setup, I do not see shortcuts like gd etc working for functions which are from other repos, even though the build artifacts are local.

This is probably similar to: https://github.com/clangd/clangd/issues/116, but how to set up this config?

**Comment by fannheyward** [comment]
As the discussion in https://github.com/clangd/clangd/issues/116, you need to setup clangd's configuration file for this, but I have no idea about how to. The client, VSCode or coc.nvim can't setup this for now https://github.com/clangd/vscode-clangd/issues/38
